### DISCUSSION TITLE
tips/02short_links

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#483] [Announcements]
# tips/02short_links

如何实现短链 本文摘自个人公众号《树哥聊编程》，主要分享了短链接服务的设置思路。@qiuchi 短链的价值 网址大家都知道，很长的一串字符串，很多时候我们还会在后面添加非常多的参数，用来便于做数据统计。下面就是微信公众号一篇文章的地址，可以看到其特别长，估计将近有几百个字符。 https://mp.weixin.qq.com/s?__biz=MzA4MjIyNTY0MQ==&mid=2647743787&idx=1&sn=1caec8eb1b81d6ee5dd7ba7fa05ac0f1&chksm=87ad0dadb0da84bb7beb5e4373a14e89fba1130c1bd2a51f4baa8021ec0abe496ce94603b6b4&token=894028224&lang=zh_CN#rd

https://qiuchixiansheng.site/tips/02short_links.html

**Comment by Scrates1** [comment]
来实验一下评论功能
